group.py

```diff
@@ -1,5 +1,12 @@
 """An example of how to represent a group of acquaintances in Python."""
 
-# Your code to go here...
 
-my_group =
```

Comment:
Be sure to include new lines to make it easier to read/ understand between each item within the group!